app/component/search/search.scss

```diff
@@ -132,11 +132,13 @@ ul#react-whatever-suggest {
 }
 
 .geolocation-bar-location-icon {
-  font-size: $search-form-location-icon-size;
+  height:16px;
+  margin-right: 10px;
 }
 
-.address-box .geolocation-bar-location-icon svg.icon {
+.geolocation-bar-location-icon svg.icon {
   fill: $hilight-color;
+  padding-top: 2px;
 }
 
 div.search-modal {
```

Comment:
We should refactor this css file so that we have:

.search-form {
...
}

.search-modal {
...
}

And we would then use SASS nesting for those elements. Now we have 
.geolocation
id:clear-input
.geolocation-selected

etc. and it's not clear how these are nested.